.github/workflows/node-zxc-build-release-artifact.yaml

```
@@ -226,7 +226,7 @@ jobs:
         uses: actions/cache@88522ab9f39a2ea568f7027eddc7d8d8bc9d59c8 # pin@v3
         if: ${{ inputs.dry-run-enabled != true && !cancelled() && !failure() }}
         with:
-          path: ${{ env.HOME }}/artifact-build
+          path: ~/artifact-build
```

The `${{ env.HOME }}` substitution is prefered because not all Github action steps properly expand the tilde `~` to be the home directory. 

```suggestion
          path: ${{ env.HOME }}/artifact-build
```

---

This substitution didn't work in this case, so i had to switch to `~`